.docs/user-guide/usage.md

```diff
@@ -0,0 +1,210 @@
+# Usage
+Report status, Initiate storage operations
+
+---
+
+## Summary
+General format:
+
+```sh
+$ rexray [option flags] [command]
+```
+
+Get general or command specific help
+
+```sh
+$ rexray --help
+```
+
+```sh
+$ rexray [command] --help
+```
+
+## Global Option Flags
+These can be invoked with all commands
+
+| Flag | Alternate form | Description |
+| --- | --- | --- |
+| `-c` | `--config=""` | Path to a custom REX-Ray configuration file |
+| `-h` | `--host=""` | designate libStorage host to execute command |
+| `-l` | `--logLevel=""` | error, warn, info, debug |
+| `-s` | `--service=""` | designate specific libStorage service command pertains to |
+| `-v` | `--verbose=true` | emit verbose information to console |
+
+## Commands
```

Comment:
Hi @cantbewong,

Please ensure there is text beneath headings instead of just tabular information or list data. All headings should be followed by some short, descriptive piece of information about that section's content.